test/color_test.dart

```diff
@@ -0,0 +1,18 @@
+library color_test;
```

Comment:
please remove the library directive and include the standard Dart copyright header.

Thanks!

Reply:
I'll include the copyright header however please confirm the removal of the library directive. Other files have similar header:

https://github.com/dart-lang/csslib/blob/11d4cf4c292c82245ec40a45671fc4e8334336c7/test/declaration_test.dart#L5

https://github.com/dart-lang/csslib/blob/11d4cf4c292c82245ec40a45671fc4e8334336c7/test/error_test.dart#L5

Reply:
Some older libraries still have them but we avoid adding them in new files. I've gone ahead and pushed the cleanup for that bit for expediency. Sorry for the delays.